lib/plumed/Install.py

```diff
@@ -163,7 +163,7 @@ def geturl(url,fname):
     os.remove("includelink")
   if os.path.isfile("liblink") or os.path.islink("liblink"):
     os.remove("liblink")
-  cmd = 'ln -s "%s/plumed2/include" includelink' % homepath
+  cmd = 'ln -s "%s/plumed2/include/plumed/wrapper" includelink' % homepath
```

Comment:
I don't like that, I think we should include `Plumed.h` with `#include "plumed/wrapper/Plumed.h"`, this is pretty much what plumed2's pkgconfig output suggests:
```
$ pkg-config --cflags plumed
-I/home/christoph/123/include
```
In the current solution we have the `plumed/wrapper` sub-directory hard-coded 3 times (once here and twice in `CMakeLists.txt`).

Reply:
Fix in 0445076

Reply:
@junghans  please review this PR *after* the work_in_progress label has been removed. And  particularly do not push changes pushed into a PR you an not assigned to. This branch is currently assigned to me, so - according to our github development policies - only I or the PR submitter may push changes to the branch. If you want to add changes to PR branches, please ask the assignee to (temporarily) assign the pull request to you (and hand it back afterwards) or grab the PR when nobody is assigned. Thanks for understanding and respecting our policies. Axel.

Reply:
@akohlmey sure, I will spend some time reading the policy in #1181 next.

Reply:
@junghans that document is for people submitting pull requests. the policies i was referring to, are the agreements between the core LAMMPS developers, i.e. steve, aidan, stan, richard, and me. they are currently not written down, and we recently changed things up a bit, in order to avoid problems that occasionally turned up. so these policies are also work in progress as this is a collaborative learning process, where we have to balance each other's skills and knowledge to meet the agreed upon development goals and efficiency in maintenance. by only having the assignee of a pull request push changes to the branch we make sure we're not stepping on each other's feet when working with an (external) contributor. we try to keep all communication on the pull request, but that is not always possible, or there are sometimes discussions about how to proceed in private e-mails, before the development moves to github.
by prematurely adding or requesting changes, you may needlessly derail the ongoing process, especially when the external contributor is not as much a git and github guru as we are. again, thanks in advance for your consideration.  axel.

Reply:
@akohlmey sure, will do, but you must admit it is petty hard to follow a policy I have never heard about! Anyhow, let's not derail this PR any further.

Reply:
@junghans i understood that and that is why i was explaining it to you. It wasn't much of a problem in the past, since you stayed out of things anyway unless you were asked to help out or you stepped in when nobody else was working on a PR. Also, like i mentioned, we are still trying to figure things out among us. things were different two months ago from where we are now, and this is not the end. It is an awkward situation, anyway, since you own the project on github without being part of the core group of developers. i don't think there are many github projects like that.

Reply:
@akohlmey: No worries, just downgrade me to a member if that helps.